doc/supported-platforms.md

```diff
@@ -12,6 +12,7 @@ Ignition is currently only supported for the following platforms:
 * [QEMU] - Ignition will read its configuration from the 'opt/com.coreos/config' key on the QEMU Firmware Configuration Device.
 * [DigitalOcean] - Ignition will read its configuration from the droplet userdata. SSH keys and network configuration are handled by coreos-metadata.
 * [Oracle Cloud] - Ignition will read its configuration from the instance userdata. SSH keys are handled by coreos-metadata.
+* [Vagrant] with Virtualbox - A vagrant plugin to generate and mount a partitioned drive for Ignition to use
```

Comment:
Keeping with the other descriptions, I would instead say something like: Vagrant on VirtualBox - Ignition will read its configuration from the config-drive generated by the vagrant-ignition plugin.